Assets/New Terrain 1.asset.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
Terrains in the Assets folder are ignored because the packaging system won't pick them up. They need to be moved into the project folder `WizardsCode/Character`

Reply:
moved to _nathan/